rllib/models/catalog.py

```diff
@@ -81,14 +81,17 @@
     # === Options for custom models ===
     # Name of a custom model to use
     "custom_model": None,
+    # Extra options to pass to the custom classes.
+    # These will be available in the Model's
+    "custom_model_config": {},
```

Comment:
Nice change! Can you also update the rst documentation by grepping for any mention of custom options?

Reply:
+1

Reply:
done